web/styles/dark_theme.css

```diff
@@ -165,18 +165,18 @@
     .group-row {
         /* This is for coluring the plus sign SVGs in stream-list UI */
         /* public */
-        .check:not(.checked) svg {
-            fill: hsl(218deg 14% 33%);
+        .check:not(.checked) .sub-unsub-icon {
+            color: hsl(218deg 14% 33%);
         }
 
         /* private */
-        .disabled svg {
+        .disabled .sub-unsub-icon {
             opacity: 0.5;
         }
 
         /* public :hover */
-        .check:not(.checked, .disabled):hover svg {
-            fill: hsl(230deg 11% 67%);
+        .check:not(.checked, .disabled):hover .sub-unsub-icon {
+            color: hsl(230deg 11% 67%);
```

Comment:
Can we also include a switch to `light-dark` to delete this section from `dark_theme.css` while we're touching it?

Reply:
I've already done that in 02af59d5393b8c86ba6318afc806b34542c60313 of #35913. I was thinking it might be good to get all those changes reviewed together instead of here.